src/version/sagas/feedsRssSagas.js

```diff
@@ -0,0 +1,24 @@
+import { takeEvery, put } from 'redux-saga/effects';
+
+import { FEEDS_RSS_ACTION, feedsRssAction } from "../actions/feedsRssAction";
+import { getRssFeedsListFromXML } from "../plugins/Feeds/rss-helper";
+
+const CORS_PROXY = "https://cors-anywhere.herokuapp.com/";
+
+export default takeEvery(FEEDS_RSS_ACTION.REQUEST, function*(action) {
+    const feedsUrl = action.rssFeedUrl;
+    const sourceId = action.sourceId;
+    const url = CORS_PROXY + feedsUrl.replace("http://", "").replace("https://", "");
```

Comment:
weird, I'd expect that you urlEncode the `feedsUrl`

Reply:
Done